manage.py

```diff
@@ -0,0 +1,95 @@
+"""
+This module is to suffice all the major commands in one script
+directly through terminal/command-line using:
+
+python manage.py <cmd> [args]
+
+cmd
+- To run the app = runserver
+    eg. python manage.py runserver
+- To create first user (by default with admin status) = createsuperuser
+    eg. python manage.py createsuperuser
+- To run all tests = test
+    eg. python manage.py test
+- To run specific test = test [args]
+    eg. python manage.py test admin tasks
+    (it will run all tests for admin and tasks)
+"""
+
+import sys
+import time
+import unittest
+
+from unittest.loader import TestLoader
+from app.database.models.user import UserModel
+from run import application
+
+
+def build(cmd_line_arguments):
+    """
+    build the app
+    """
+    if len(cmd_line_arguments) > 1:
+        if cmd_line_arguments[1] == "runserver":
+            run_server()
+        elif cmd_line_arguments[1] == "createsuperuser":
+            create_superuser()
+        elif cmd_line_arguments[1] == "test":
+            test(cmd_line_arguments[2:])
+        else:
+            print("'" + cmd_line_arguments[1]  + "'" + " command is not available!")
+
+    else:
+        print(" No parameters given ")
+
+def run_server():
+    """
+    It is used to run the app.
+    """
+    application.run(port=5000)
+    return 0
+
+def test(list_to_test):
+    """
+    It will perform tests (on all or some specific parts)
+    """
+    # if additional arguments then it will test only for specified parts
+    testloader = TestLoader()
+    if list_to_test:
+        for test_area in list_to_test:
+            suite = testloader.discover(start_dir=test_area, top_level_dir="tests/")
+            print("\nTesting " + test_area)
+            unittest.TextTestRunner().run(suite)
+
+    else:
+        suite = testloader.discover(start_dir=".", top_level_dir=None)
+        unittest.TextTestRunner().run(suite)
+    return 0
+
+def create_superuser():
+    """
+    It will create the admin (if it is first user)
+    """
+    # creating new user without
+    with application.app_context():
+        if UserModel.is_empty():
+            heading = "-"*26 + " Registering SuperUser(First user) " + "-"*26
+            print(heading)
+            name = input("Enter Name: ")
+            username = input("Enter Username: ")
+            email = input("Enter Email: ")
```

Comment:
the email and password should be environment variables instead of input, I think. Because this will be an by Travis CI, and ti would be easier to setup like the other variables. Can you see how other environment variables are imported and handled on `config.py`. Not sure about the other attributes 🤔

Reply:
@isabelcosta I will see to that

Reply:
@isabelcosta we can do that using environment variables but in my opinion it is user friendly to have it as an input parameter for user when creating first(superuser). Like we have in Django

Reply:
and for using travis we can send testing input in a text file.
For example: https://stackoverflow.com/questions/37107609/how-to-test-a-project-that-need-keyboard-input-with-travis-ci-org